Formula/r/rust.rb

```diff
@@ -36,6 +36,9 @@ class Rust < Formula
   depends_on "cmake" => :build
   depends_on "ninja" => :build
   depends_on "python@3.11" => :build
+  depends_on "libgit2"
+  depends_on "libssh2"
+  depends_on "llvm"
```

Comment:
Dependency loop on Linux: `llvm -> swig (build) -> ruby (test) -> rust (build)`

Reply:
Can probably use `python` instead of `ruby` to test `swig`. Will look into that.

Reply:
#140437